pandasai/agent/base.py

```diff
@@ -98,6 +98,19 @@ def __init__(
             self._vectorstore = BambooVectorStore(logger=self.logger)
             self.context.vectorstore = self._vectorstore
 
+        elif self._vectorstore == "lancedb" and os.environ.get("PANDASAI_API_KEY"):
```

Comment:
@PrashantDixit0 We don't need these lines to add lancedb check here BambooVectorStore is for default if no vectorstore is provided. The Constructor except vectorstore object not name.

Reply:
Sure :+1: